pages.fr/common/z.md

```diff
@@ -0,0 +1,28 @@
+# z
+
+> Recherche les répertoires les plus utilisés et permet une navigation rapide à l'aide de chaine de caractères ou de regex.
+> Plus d'informations: <https://github.com/rupa/z>.
+
+- Aller dans un répertoire qui contient "foo" dans son nom:
```

Comment:
I remember that the French language use a space before `:`. Please fix all the example descriptions.
```suggestion
- Aller dans un répertoire qui contient "foo" dans son nom :
```